trlc/lexer.py

```diff
@@ -108,6 +108,20 @@ def context_lines(self):
         return [stripped_line,
                 " " * stripped_offset + "^" * min(tlen, maxtrail)]
 
+    def get_end_location(self):
+        """Get location pointing to the end of a source."""
+
+        lines_in_between = self.lexer.content[self.start_pos : self.end_pos + 1].count("\n")
+        end_line = self.line_no + lines_in_between
+        n = self.end_pos
+        while n > self.start_pos:
```

Comment:
Not sure this logic fully works. For this input file:
```trlc
'''foo
bar'''
```

We would get an end location of line 2, column 7, but column should be 6.